gui/default/syncthing/core/syncthingController.js

```diff
@@ -670,12 +647,12 @@ angular.module('syncthing.core')
             refreshNeed($scope.neededFolder);
         };
 
-        $scope.failedPageChanged = function (page) {
-            $scope.failedCurrentPage = page;
-        };
-
-        $scope.failedChangePageSize = function (perpage) {
-            $scope.failedPageSize = perpage;
+        $scope.refreshFailed = function (page, perpage) {
```

Comment:
Is the capitalization if perpage consistent in JS and rest of Go?

Reply:
Yes, it's everywhere like this. Maybe it is like this because the url parameter is lowercase.